lib/tune.cpp

```diff
@@ -678,6 +680,35 @@ namespace quda
 
   static TimeProfile launchTimer("tuneLaunch");
 
+  using tunepair = std::pair<float, TuneParam>;
+  struct tunepaircomp {
+    bool operator()(const tunepair &left, const tunepair &right) const { return (left.first) < (right.first); }
+  };
+
+  class TuneCandidates : public std::priority_queue<tunepair, std::vector<tunepair>, tunepaircomp>
+  {
+  private:
+    static constexpr size_t maxSize = 5;
```

Comment:
style comment: prefer `max_size`, I think we should be moving away from camel case, especially for member variables (I'll _tolerate_ it for member functions).

Reply:
good point.

Reply:
86b46b64e4871473715e683dfc1ab46d5de8d18c